tests/Behat/Context/LoginContext.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\MFA\Tests\Behat\Context;
+
+use SilverStripe\CMS\Tests\Behaviour\LoginContext as CMSLoginContext;
+use SilverStripe\MFA\Extension\SiteConfigExtension;
+use SilverStripe\SiteConfig\SiteConfig;
+
+/**
+ * Overridden from the CMS module to ensure that MFA can be handled during fixtured member generation
+ */
+class LoginContext extends CMSLoginContext
+{
+    public function iAmLoggedInWithPermissions($permCode)
+    {
+        // Set MFA to optional, perform login logic, then skip MFA
+        $this->multiFactorAuthenticationIsOptional();
+        parent::iAmLoggedInWithPermissions($permCode);
+
+        // Wait for MFA to load
+        $this->getMainContext()->getSession()
+            ->wait(5000, 'document.getElementsByClassName("mfa-app-title").length === 1');
```

Comment:
Is there not another method (named `spin` or something) that is better than waiting an arbitrary `5` seconds for something to complete?
I think @ScopeyNZ has done things along this line before.

Reply:
This is waiting _up to_ five seconds for that JavaScript condition to be true. We'd expect it to resolve much faster than that.